src/psyclone/psyir/transformations/omp_target_trans.py

```diff
@@ -86,12 +87,34 @@ class OMPTargetTrans(RegionTrans):
     '''
     excluded_node_types = (CodeBlock, )
 
+    def validate(self, node, options=None):
+        # pylint: disable=signature-differs
+        '''
+        Check that we can safely enclose the supplied node or list of nodes
+        within OpenACC kernels ... end kernels directives.
```

Comment:
this isn't a 'kernels' transformation! :-)

Reply:
Ups, copy-pasting mistake. Fixed now.